packages/prisma/migrations/20240209023519_add_document_audit_logs/migration.sql

```diff
@@ -6,7 +6,13 @@
 
 */
 -- AlterTable
-ALTER TABLE "Field" ADD COLUMN     "secondaryId" TEXT NOT NULL;
+ALTER TABLE "Field" ADD COLUMN     "secondaryId" TEXT;
+
+-- Set all null secondaryId fields to a uuid
+UPDATE "Field" SET "secondaryId" = gen_random_uuid()::text WHERE "secondaryId" IS NULL;
+
+-- Restrict the Field to required
+ALTER TABLE "Field" ALTER COLUMN "secondaryId" SET NOT NULL;
```

Comment:
@Mythie could you please have a look at this custom migration change to backfill `secondaryId`

We can't use cuid in migrations, so I'm using UUID

This means that we're going to have two different types of IDs for `secondaryId`, is that fine?

Reply:
That's fine, lgtm